app/models/vm_or_template/operations/relocation.rb

```diff
@@ -3,6 +3,8 @@ module VmOrTemplate::Operations::Relocation
 
   included do
     supports_not :live_migrate, :reason => _("Live Migrate VM Operation is not available for VM or Template.")
+    supports_not :evacuate,
+                 :reason => _("Evacuate VM Operation is not available for VM or Template.")
```

Comment:
I dont like the phrasing of this. It's talking about "Evacuate **VM**" and  "or **Template**"

Can you rename it to `_("Operation not supported")` and the above for `live_migrate` too?
Because in the end this only pops up as alt text for buttons - if at all. And like this we dont have to translate (i18n) redundant messages.